webapp/channels/src/components/preparing_workspace/invite_members.tsx

```diff
@@ -242,4 +237,15 @@ const InviteMembers = (props: Props) => {
     );
 };
 
+const messages = defineMessages({
```

Comment:
Not only for this file, but I like better things to be defined before the component (everything is defined before it is used). But 0/5 (mostly a personal preference).

Reply:
I'm back to defining it after because I care about reading the component more than the strings. The only reason that this one is before is because it's part of the `defaultProps` which needs `messages` to be defined before the component is